daemon/container_operations_windows.go

```diff
@@ -35,6 +43,62 @@ func detachMounted(path string) error {
 	return nil
 }
 
+func (daemon *Daemon) setupSecretDir(c *container.Container) (setupErr error) {
+	if len(c.SecretReferences) == 0 {
+		return nil
+	}
+
+	localMountPath := c.SecretMountPath()
```

Comment:
This is copied from `container_operations_unix.go` and I prefer to keep the files in sync initially.